### PR TITLE
Remove travis artifacts

### PR DESCRIPTION
**Pull request by sobolevn**
### Changes

Travis CI is long gone. It is not used anymore.
So:
- [x] I've removed `.travis.yml`
- [x] I've added a new badge for GitHub Actions to the README
- [x] I've removed an outdated comment  

### Checklist

I don't think that this should be tested / or added to changelog.

- [ ] Added markup tests, or they don't apply here because...
- [ ] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,7 @@
 [![NPM downloads weekly](https://badgen.net/npm/dw/highlight.js?label=npm+downloads&color=purple)](https://www.npmjs.com/package/highlight.js)
 [![jsDelivr CDN downloads](https://badgen.net/jsdelivr/hits/gh/highlightjs/cdn-release?label=jsDelivr+CDN&color=purple)](https://www.jsdelivr.com/package/gh/highlightjs/cdn-release)
 
-<!-- [![build and CI status](https://badgen.net/github/checks/highlightjs/highlight.js?label=build)](https://github.com/highlightjs/highlight.js/actions?query=workflow%3A%22Node.js+CI%22) -->
-![build and CI status](https://badgen.net/github/checks/highlightjs/highlight.js/main?label=build)
+[![ci status](https://github.com/highlightjs/highlight.js/actions/workflows/tests.js.yml/badge.svg?event=push)](https://github.com/highlightjs/highlight.js/actions/workflows/tests.js.yml)
```
Comment (joshgoebel) [inline README.md:10]: Was there some issue with the badgen badge?  We use the same source for all the badges for consistency.

Comment (sobolevn) [reply README.md:10]: I don't know: I coppied the one that github officially provides.

Comment (joshgoebel) [reply README.md:10]: I'm ok with adding the link, but lets go back to using the previous badge - then this would be mergable.

Thanks!

Comment (sobolevn) [inline README.md:10]:
```suggestion
[![ci status](https://badgen.net/github/checks/highlightjs/highlight.js/main?label=build)](https://github.com/highlightjs/highlight.js/actions/workflows/tests.js.yml)
```

**Comment by sobolevn** [comment]
Sure thing, done!

**Review by joshgoebel — APPROVED** [review]

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**